Sail/src/API/Audio/AudioEngine.cpp

```diff
@@ -632,7 +634,7 @@ HRESULT AudioEngine::initXAudio2() {
 	UINT32 flags = XAUDIO2_1024_QUANTUM;
 
 #ifdef DEVELOPMENT
-	flags |= XAUDIO2_DEBUG_ENGINE;
+	//flags |= XAUDIO2_DEBUG_ENGINE;
 #endif
```

Comment:
Is it necessary to #ifdef when not doing anything in the loop?

Reply:
Yes box